app/src/main/java/com/github/se/cyrcle/ui/parkingDetails/ParkingDetailsAlertDialogShowImage.kt.kt

```diff
@@ -29,9 +33,12 @@ import com.github.se.cyrcle.ui.theme.atoms.IconButton
  */
 @Composable
 @OptIn(ExperimentalMaterial3Api::class)
-fun ParkingDetailsAlertDialogShowImage(onDismiss: () -> Unit, imageUrl: String) {
+fun ParkingDetailsAlertDialogShowImage(
```

Comment:
you forgot to update the documentation here.

Reply:
Done, thanks!